src/components/skills-quiz/SearchJobCard.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import Truncate from 'react-truncate';
+import Skeleton from 'react-loading-skeleton';
+// import { camelCaseObject } from '@edx/frontend-platform/utils';
```

Comment:
nit: remove the commented out import.

Reply:
This piece of code will be required once we have jobs data coming as hits form Algolia. I will add a TODO above this comment to mention that.